### ISSUE TITLE
Make harp diagram markup play nice

### ISSUE THREAD

**Issue by josiah-wolf-oberholtzer** [Abjad/abjad#742]
Currently the following is parsed corrected, but the quotes are dropped when formatting:

```
markup = Markup(r'\harp-pedal #"^--|v-v-")
```

**Comment by trevorbaca** [comment]
Still valid in June 2017:
```
>>> Markup(r'\harp-pedal #"^--|v-v-")
  File "<stdin>", line 1
    Markup(r'\harp-pedal #"^--|v-v-")
                                    ^
```

**Comment by trevorbaca** [comment]
Fixed in Abjad 3.1; set `literal=True` in `abjad.Markup`.

```
markup = abjad.Markup(
    r'\markup { \harp-pedal #"^--|v-v-" }',
    literal=True,
    )
abjad.f(markup)
note = abjad.Note("c'4")
abjad.attach(markup, note)
abjad.f(note)
```

<img width="247" alt="harp-pedal" src="https://user-images.githubusercontent.com/5804733/52541821-ceba4380-2d5e-11e9-8813-3ebf983bca8b.png">
